revolv/templates/base/partials/foot.html

```diff
@@ -73,6 +73,7 @@
         <ul class="social-ul">
           <li><a href="https://www.facebook.com/solarseedfund/" target="_blank" class="icon-fb icons"></a></li>
           <li><a href="https://twitter.com/RE_volv" target="_blank" class="icon-tw icons"></a></li>
+          <li><a href="#" target="_blank" class="icon-g-plus icons"></a></li>
```

Comment:
This link doesn't seem to do anything (it's a link to `#`, which will just open up the current page in a new tab). Is it necessary?